# 裁判警告

Header header

# 装甲ID
int32 armorID

# 伤害类型
# 装甲伤害
uint8 HURT_ARMOR_DAMAGE = 0
# 模块掉线
uint8 HURT_MODULE_OFFLINE = 1
# 射击超射速
uint8 HURT_SHOOTER_SPEED_LIMIT = 2
# 射击超热量
uint8 HURT_SHOOTER_HEAT_LIMIT = 3
# 底盘超功率
uint8 HURT_CHASSIS_POWER_LIMIT = 4
# 底盘超功率
uint8 HURT_ARMOR_COLLISION = 5
uint8 type